ftw/theming/resources/scss/globals/helpers.scss

```diff
@@ -74,6 +75,22 @@
   }
 }
 
+@mixin link-color($color-link: $color-link, $color-link-hover: $color-link-hover) {
```

Comment:
The whole mixin structure is build upon this behavior that the variables are scoped. So if this would not word I would be really fucked up 😂 . So in my opinion this should not be an issue.